### ISSUE TITLE
Configure warnings, partclone 0.3.17

### ISSUE THREAD

**Issue by Andy2051** [Thomas-Tsai/partclone#155]
Common to all filesystems:

`configure: WARNING: *** tinfo library (libtinfo) not found`

which comes from:
```
AC_CHECK_LIB([tinfo], [initscr], tinfo=1,
    AC_MSG_WARN([*** tinfo library (libtinfo) not found]))
```
libtinfo5 and libtinfo-dev installed

Specific to ntfs:

`configure: WARNING: *** NTFS(ntfsprogs) header not found`

coming from

```
    AC_CHECK_HEADERS([ntfs/version.h], ntfsprogs_h=1,
    AC_MSG_WARN([*** NTFS(ntfsprogs) header not found]))
```
ntfs-3g-dev installed, there is no ntfs folder in /usr/include and no version.h in /usr/include/ntfs-3g

Host system - Linux mint 19.0 and Linux mint 20.0.

**Comment by Thomas-Tsai** [comment]
partclone depends on ntfs-3g-dev. The easy command is ```apt-get build-dep partclone ``` which help you install some basic libraries.

**Comment by Andy2051** [comment]
> partclone depends on ntfs-3g-dev

ntfs-3g-dev (1:2017.3.23-2ubuntu0.18.04.2) is installed as I said in my first post.

The content is:
`/.
/usr
/usr/bin
/usr/bin/ntfsck
/usr/bin/ntfsdump_logfile
/usr/bin/ntfsmftalloc
/usr/include
/usr/include/ntfs-3g
/usr/include/ntfs-3g/acls.h
/usr/include/ntfs-3g/attrib.h
/usr/include/ntfs-3g/attrlist.h
/usr/include/ntfs-3g/bitmap.h
/usr/include/ntfs-3g/bootsect.h
/usr/include/ntfs-3g/cache.h
/usr/include/ntfs-3g/collate.h
/usr/include/ntfs-3g/compat.h
/usr/include/ntfs-3g/compress.h
/usr/include/ntfs-3g/debug.h
/usr/include/ntfs-3g/device.h
/usr/include/ntfs-3g/device_io.h
/usr/include/ntfs-3g/dir.h
/usr/include/ntfs-3g/ea.h
/usr/include/ntfs-3g/efs.h
/usr/include/ntfs-3g/endians.h
/usr/include/ntfs-3g/index.h
/usr/include/ntfs-3g/inode.h
/usr/include/ntfs-3g/ioctl.h
/usr/include/ntfs-3g/layout.h
/usr/include/ntfs-3g/lcnalloc.h
/usr/include/ntfs-3g/logfile.h
/usr/include/ntfs-3g/logging.h
/usr/include/ntfs-3g/mft.h
/usr/include/ntfs-3g/misc.h
/usr/include/ntfs-3g/mst.h
/usr/include/ntfs-3g/ntfstime.h
/usr/include/ntfs-3g/object_id.h
/usr/include/ntfs-3g/param.h
/usr/include/ntfs-3g/plugin.h
/usr/include/ntfs-3g/realpath.h
/usr/include/ntfs-3g/reparse.h
/usr/include/ntfs-3g/runlist.h
/usr/include/ntfs-3g/security.h
/usr/include/ntfs-3g/support.h
/usr/include/ntfs-3g/types.h
/usr/include/ntfs-3g/unistr.h
/usr/include/ntfs-3g/volume.h
/usr/include/ntfs-3g/xattrs.h
/usr/lib
/usr/lib/x86_64-linux-gnu
/usr/lib/x86_64-linux-gnu/libntfs-3g.a
/usr/lib/x86_64-linux-gnu/libntfs-3g.so
/usr/lib/x86_64-linux-gnu/pkgconfig
/usr/lib/x86_64-linux-gnu/pkgconfig/libntfs-3g.pc
/usr/share
/usr/share/doc
/usr/share/doc/ntfs-3g-dev
`

**Comment by Thomas-Tsai** [comment]
yes, ntfs/version.h is for ntfs-progs test, it's for some old system only, we can skip it.

I check my ubuntu and my ntfs-3g-dev is also in /usr/include/ntfs-3g/, will check like this
dpkg -L ntfs-3g-dev | grep ntfs-3g/misc.h
/usr/include/ntfs-3g/misc.h

for libtinfo, we maybe also need libncursesw5-dev.

In my ubuntu 20.04, we just need :

sudo apt-get build-dep partclone
apt-get source partclone
cd partclone-0.3.13+dfsg/
./autogen
./configure --enable-ntfs
make

if you already installed ntfs-progs, try to remove it?!

or check the control file
https://github.com/Thomas-Tsai/partclone/blob/master/README.Packages/debian/control
